### ISSUE TITLE
Accessibility issues regarding EGit (for screen reader users)

### ISSUE THREAD

**Issue by hamidrezaabroshan** [eclipse-jdt/.github#19]
I am blind and use computers with the help of screan-reader NVDA. I
have some accessibility issues while using EGit.
1. when having merge conflicts, and resolving, in the merge tool, when
navigating on the list of files it is not distinguishable if each of
them is resolved or not yet.
They should be marked by a prefix like a * till screen-readers can report.
2. To study and comparing codes is difficult for a blind user that is
not able to study left and right editors in merge tools
simultaneously; so I prefer to study the code in a single normal
eclipse code editor that determines conflicts by <<< >>>.
when I need to open a file code to resolving a conflict it is better
to be a menu item like "open in code editor" in the context menu. I
know, the selected file will be opened in code editor if I select
""open from the context menu then select "project Explorer" option and
next I open code editor from there but this routine is too long. It
can have a shortcut or another shorter way.)
Thanks.

**Comment by fapdash** [comment]
@hamidrezaabroshan Can you please report this to the bug tracker of EGit over here: https://bugs.eclipse.org/bugs/enter_bug.cgi?product=EGit
You'll need an https://www.eclipse.org/ account to sign in to the Eclipse Bugzilla.

**Comment by hamidrezaabroshan** [comment]
Thanks @fapdash  reported to EGit project.
https://bugs.eclipse.org/bugs/show_bug.cgi?id=580964
but it is not specified clearly in bugzilla page that which bug should be reported to which project and also finding the correct link is hard 
If it helps, I'll create another issue regarding this.

**Comment by jjohnstn** [comment]
Closing this issue as it is not a JDT problem and has been re-routed to EGit by the reporter.
